static/css/section/_desktop.scss

```diff
@@ -1,258 +1,153 @@
-// @section desktop-home
-.row-case-studies {
-  background-color: $dark-aubergine;
-  background-image: url('#{$asset-server}3c1c3b37-background-tile.jpg');
-  color: #fff;
-
-  .pull-quote {
-    margin-bottom: .75em;
-
-    p {
-      color: #fff;
-      font-size: 1em;
-      line-height: 1.5em;
+@charset 'UTF-8';
+
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Cloud section
+//  Author:			Web Team at Canonical Ltd
+//  Last edited by: Peter Mahnke
+//  Last edited on: 18 September 2016
+//
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+
+// @global desktop section_title
+.desktop {
+
+  .row-case-studies {
+    background-color: $dark-aubergine;
+    background-image: url('#{$asset-server}3c1c3b37-background-tile.jpg');
+    color: $header-link-color;
 
-      span {
-        top: .2em;
+    blockquote {
+      @media only screen and (min-width: $breakpoint-medium) {
+        margin-bottom: 20px;
       }
     }
 
-    cite {
-      color: #fff;
-      font-size: 1em;
-      line-height: 1.5em;
-    }
-  }
-
-  @media only screen and (min-width: $breakpoint-medium) {
-    > h2:first-child {
-      margin-bottom: 71%;
-    }
-  }
-}
+    .pull-quote {
+      margin-bottom: .75em;
 
-/// small screens
-@media only screen and (max-width: $breakpoint-medium) {
-  .row-apps {
-    overflow: hidden;
-    width: 100%;
+      p {
+        color: $header-link-color;
+        font-size: 1em;
+        line-height: 1.5em;
 
-    .one-col {
-      float: left;
-      width: 20%;
-      margin-right: 2.21238%;
-    }
+        span {
+          top: .2em;
+        }
+      }
 
-    .three-col {
-      margin-right: 0;
-      float: left;
-      width: 64%;
+      cite {
+        color: $header-link-color;
+        font-size: 1em;
+        line-height: 1.5em;
+      }
     }
-  }
-}
-
-.row-landscape,
-.row-complete {
-  overflow: hidden;
-  padding-bottom: 20px;
-
-  @media only screen and (min-width: $breakpoint-medium) {
-    padding-bottom: 0;
 
-    .pull-right {
-      max-width: 60%;
+    @media only screen and (min-width: $breakpoint-medium) {
+      > h2:first-child {
```

Comment:
This seems very specific. I wonder if the mark up could be improved or improved by adding a descriptive class?